src/strings/en-us.json

```diff
@@ -1459,7 +1459,7 @@
     "TitleHardwareAcceleration": "Hardware Acceleration",
     "TitleHostingSettings": "Hosting Settings",
     "TitlePlayback": "Playback",
-    "TonemappingAlgorithmHelp": "Tone mapping can be fine-tuned. If you are not familiar with these options, just keep the default. The recommended value is Hable.",
+    "TonemappingAlgorithmHelp": "Tone mapping can be fine-tuned. If you are not familiar with these options, just keep the default. The recommended value is BT.2390.",
```

Comment:
Since the recommended value changed, should the default value also be changed?

Reply:
There should be no default value for tone-mapping algorithm. In theory, the SDR version of each HDR movie has a different algorithm. We recommend BT.2390 to follow the default behavior of mpv.

Reply:
I get what you're saying, but when I opened the webpage on a brand new Jellyfin build, I had the tone-mapping set to "Hable", which will be saved to the server and then used.. I think the default in the drop down should match what the strings say

Reply:
Have you upgraded the server to master branch?
The default value is set on the server side.

https://github.com/jellyfin/jellyfin/blob/78bb581f0c7d27f7f9a7a6d3ac81a2a6c6fbe781/MediaBrowser.Model/Configuration/EncodingOptions.cs#L21

Reply:
you're 100% correct, I needed to re-re-clean my config. Works great when starting from scratch